packages/runtime/container-runtime/package.json

```diff
@@ -112,6 +112,10 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
```

Comment:
From what I have heard, any change that updates this should go in next. There have been unexpected errors because of these changes.

Reply:
Reverted